source/nodejs/adaptivecards-site/package.json

```diff
@@ -11,7 +11,7 @@
     "start": "hexo server"
   },
   "hexo": {
-    "version": "5.2.0"
+    "version": "4.2.1"
```

Comment:
don't think you meant to change this :)

Reply:
I don't, but then help me understand how it happened - I litterally just compiled (npx lerna run build)

Reply:
not sure really -- might be something hexo modified when you were testing your changes?